gaia/parser.py

```diff
@@ -35,12 +40,29 @@
     gaia.geo.geo_inputs, inspect.isclass) if x[0].endswith('IO')])
 
 
+def load_plugins():
```

Comment:
@aashish24 I can move the load_plugins() definition to gaia/__init__.py or gaia/core.py, do you have a preference?

The raster and vector parts of geo seem more like core functionality, but I think some of it could be split up into multiple plugins, for instance:
  - twitter plugin
  - stats plugin (anything that depends on the pysal module) 
  - maybe a raster plugin too, but keep the vector io's/processes in geo as core functionality?